### PR TITLE
Wire-up featureService namespace filter

### PR DESCRIPTION
**Pull request by c-w**
As per conversation with @erikschlegel.

### CODE REVIEW

**Review by jcjimenez — APPROVED** [review]
LGTM +1 awyeah

**Comment by c-w** [comment]
Going to wait on a review from @kevinhartman since this touches the transform context code.

**Review by kevinhartman — APPROVED** [review]

**Pull request merged**
